tox.ini

```
@@ -6,8 +6,8 @@ envlist =
     py37-checkqa
     docs
     py{37,py37}-dj{32}
-    py{38,py38,39,py39,310}-dj{32,40,41}
-    py{311}-dj{41}
+    py{38,py38,39,py39,310}-dj{32,40,41,42}
+    py{311}-dj{41,42}
```

```suggestion
    py{38,py38,39,py39,310}-dj{32,40,41,42,main}
    py{311}-dj{41,42,main}
```
I removed Django main support precisely because of the bug you fixed with this PR, could you add it back so we can verify your fix also with the code of the main Django branch?